lago/prefix.py

```diff
@@ -645,7 +645,10 @@ def _handle_lago_template(
             template_store.get_stored_metadata(template_version, ),
         )
         base = template_store.get_path(template_version)
-        qemu_cmd = ['qemu-img', 'create', '-f', 'qcow2', '-b', base, disk_path]
+        qemu_cmd = [
+            'qemu-img', 'create', '-f', 'qcow2', '-o', 'compat=1.1', '-o',
+            'lazy_refcounts=on', '-b', base, disk_path
+        ]
```

Comment:
I don't remember exactly why, but I could not change the cache mode. I'll try to look into it later.